src/Tools/IiifHelper.php

```diff
@@ -76,12 +80,21 @@ public function getRemoteIIIFImageInfo($remoteUrl) {
   }
 
   public function getImageSizes() {
-    if (empty($this->remoteInfoJson)) {
+    if (empty($this->remoteInfoJsonData)) {
       return [];
     }
-    if (isset($this->remoteInfoJson['sizes'])) {
-      return $this->remoteInfoJson['sizes'];
+    if (isset($this->remoteInfoJsonData['sizes'])) {
+      return $this->remoteInfoJsonData['sizes'];
     }
     return [];
   }
+
+  public function getInternalInfoJson() {
```

Comment:
Here is where we can force people to pass the identifier (e.g) instead of having it set in the constructor